### PR TITLE
Docs: Adding Metapilot Section

### PR DESCRIPTION
**Pull request by RounakDhillon**
### Describe your changes:

Fixes <issue-number>

I worked on adding metapilot section.

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-docs/content/v1.5.x/menu.md [hunk 1]
```diff
@@ -2089,4 +2089,12 @@ site_menu:
...
   - category: SDK / Go
     url: /sdk/go
+
+  - category: Metapilot
+    url: /metapilot
+    isCollateOnly: true
+  
+   category: Metapilot / How to Use MetaPilot
+    url: /metapilot/how-to-use-metapilot
+    isCollateOnly: true
```
Comment (ayush-shah) [inline openmetadata-docs/content/v1.5.x/menu.md:2099]: Let's remove it from `menu.md` and add it in `collate-menu.md`

**Review by ayush-shah — CHANGES_REQUESTED** [review]
One comment, also can you take care of the same comment for 1.6 as well, Thanks.

File: openmetadata-docs/content/v1.5.x/menu.md (hunk 1 above)
Comment (RounakDhillon) [reply openmetadata-docs/content/v1.5.x/menu.md:2099]: ok on it

File: openmetadata-docs/content/v1.5.x/collate-menu.md [hunk 2]
```diff
@@ -1874,4 +1874,9 @@ site_menu:
     url: /sdk/java
   - category: SDK / Go
     url: /sdk/go
+  - category: Metapilot
+    url: /metapilot
+  
+   category: Metapilot / How to Use MetaPilot
+    url: /metapilot/how-to-use-metapilot
```
Comment (ayush-shah) [inline openmetadata-docs/content/v1.5.x/collate-menu.md:1881]:
```suggestion
  - category: Metapilot / How to Use MetaPilot
    url: /metapilot/how-to-use-metapilot
```

**Review by ayush-shah — CHANGES_REQUESTED** [review]

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**